### DISCUSSION TITLE
Paid Panicone Index does not work

### DISCUSSION THREAD

**Discussion by ubiratandias** [FlowiseAI/Flowise#1894] [Q&A]
Hello everyone, all good? 
When I send PDF files through Flowise to an index started “Free tier” I have no problems.
But when uploading to a paid index I get this error when uploading files:
Error: PineconeNotFoundError: A call to https://api.pinecone.io/indexes/empresa2 returned HTTP status 404.
When I convert the same free index to a paid index it stops working.
The two indexes are in the same environment so I am using the same apikey.
Any idea?
Tks

**Comment by benoita891** [comment]
same issue here

**Reply by HenryHengZJ** [reply]
hopefully after this changes get pushed it works fine - https://github.com/FlowiseAI/Flowise/pull/2036
